velodrome/fetcher/client.go

```diff
@@ -191,9 +191,9 @@ func (client *Client) FetchIssueEvents(latest *int, c chan *github.IssueEvent) {
 	for {
 		client.limitsCheckAndWait()
 
-		events, resp, err := githubClient.Issues.ListRepositoryEvents(client.Org, client.Project, opt)
+		events, resp, err := githubClient.Issues.ListIssueEvents(client.Org, client.Project, issueID, opt)
 		if err != nil {
-			glog.Error("Request failed. Wait before trying again.")
+			glog.Errorf("Request failed (%s). Wait before trying again.", err)
```

Comment:
IMO I don't see why we need to be generic here. Can't we say "ListIssueEvents() failed?"